site/src/examples/scrim/WithChild.tsx

```diff
@@ -0,0 +1,110 @@
+import { ReactElement, useState } from "react";
+import {
+  Button,
+  Card,
+  Display3,
+  H3,
+  Scrim,
+  Spinner,
+  StackLayout,
+  Text,
+} from "@salt-ds/core";
+import { ArrowDownIcon, ArrowUpIcon } from "@salt-ds/icons";
+
+export const WithChild = (): ReactElement => {
+  const [open, setOpen] = useState(false);
+  const [buttonText, setButtonText] = useState("Show scrim");
+
+  const handleClick = () => {
+    setOpen(!open);
```

Comment:
This should use a function https://react.dev/reference/react/useState#setstate

```suggestion
    setOpen(old => !old);
```